kinder/pkg/cluster/manager/actions/kubeadm-init.go

```diff
@@ -97,9 +95,7 @@ func kubeadmInit(cp1 *status.Node, copyCertsMode CopyCertsMode, patchesDir, igno
 		)
 	}
 	if patchesDir != "" {
-		if cp1.MustKubeadmVersion().LessThan(constants.V1_22) {
-			initArgs = append(initArgs, "--experimental-patches", constants.PatchesDir)
-		}
+		initArgs = append(initArgs, "--patches", constants.PatchesDir)
```

Comment:
--patches was completely missing in some places

Reply:
IIRC, this is in the kubeadm config `/kind/kubeadm.conf` in new versions.
```
patches:
  directory: /kinder/patches
```

Reply:
right..thanks for the reminder.
only upgrade needs --patches

Reply:
updated!